pkg/plugins/plugins.go

```diff
@@ -73,33 +77,40 @@ func (pm *PluginManager) Init() error {
 	}
 
 	pm.log.Info("Starting plugin search")
+
+	pm.log.Info("Scanning main plugin directory")
 	plugDir := path.Join(setting.StaticRootPath, "app/plugins")
-	if err := pm.scan(plugDir); err != nil {
-		return errutil.Wrapf(err, "Failed to scan main plugin directory '%s'", plugDir)
+	if err := pm.scan(plugDir, false); err != nil {
+		return errutil.Wrapf(err, "failed to scan main plugin directory '%s'", plugDir)
 	}
 
-	pm.log.Info("Checking Bundled Plugins")
+	pm.log.Info("Scanning bundled plugins directory")
 	plugDir = path.Join(setting.HomePath, "plugins-bundled")
```

Comment:
Maybe want to extract this to setting package in similar manner as `setting.PluginsPath`. A new `setting.BundledPluginsPath`?

Reply:
Bit unsure of what it would entail - from the looks of it we are also moving from global variables in `setting` to fields on `setting.Cfg`?

Reply:
Yeah you can put it on `Cfg`.

Reply:
@marefr Does that mean I should also add a configuration parameter `paths.bundled_plugins` to conf/defaults.ini and conf/sample.ini?

Reply:
Only reason for doing that would be if it can vary between distributions. For example if there are special use case for deb or rpm

Reply:
@marefr OK, I added it as a field to setting.Cfg.